kubless topic create fails

Pretty sure the problem is my setup.  A more helpful error message would be welcome.

```
$ kubeless version
Kubeless version: 0.0.16 (f87348a6)
```

```
$ kubeless topic create telemetry
panic: runtime error: index out of range

goroutine 1 [running]:
main.execCommand(0xc4203c71e0, 0xb, 0xb, 0x1d3a743, 0x8)
	/home/travis/gopath/src/github.com/kubeless/kubeless/cmd/kubeless/topicCreate.go:60 +0x584
main.glob..func9(0x2aa22a0, 0xc42023d680, 0x1, 0x1)
	/home/travis/gopath/src/github.com/kubeless/kubeless/cmd/kubeless/topicCreate.go:46 +0x212
github.com/kubeless/kubeless/vendor/github.com/spf13/cobra.(*Command).execute(0x2aa22a0, 0xc42023d630, 0x1, 0x1, 0x2aa22a0, 0xc42023d630)
	/home/travis/gopath/src/github.com/kubeless/kubeless/vendor/github.com/spf13/cobra/command.go:603 +0x22b
github.com/kubeless/kubeless/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc42018f680, 0x2aa1a20, 0x2aa2080, 0x2aa2d40)
	/home/travis/gopath/src/github.com/kubeless/kubeless/vendor/github.com/spf13/cobra/command.go:689 +0x339
github.com/kubeless/kubeless/vendor/github.com/spf13/cobra.(*Command).Execute(0xc42018f680, 0xc4200001a0, 0xc4200001a0)
	/home/travis/gopath/src/github.com/kubeless/kubeless/vendor/github.com/spf13/cobra/command.go:648 +0x2b
main.main()
	/home/travis/gopath/src/github.com/kubeless/kubeless/cmd/kubeless/kubeless.go:41 +0x2f
```

--- a/cmd/kubeless/topicCreate.go
+++ b/cmd/kubeless/topicCreate.go
@@ -53,7 +53,12 @@ func execCommand(command []string, ctlNamespace string) {
 	f := cmdutil.NewFactory(nil)
 
 	k8sClientSet := utils.GetClientOutOfCluster()
-	pods, _ := utils.GetPodsByLabel(k8sClientSet, ctlNamespace, "kubeless", "kafka")
+	pods, err := utils.GetPodsByLabel(k8sClientSet, ctlNamespace, "kubeless", "kafka")
+	if err != nil {
+		logrus.Fatalf("Can't find the kafka pod: %v", err)
+	} else if len(pods.Items) == 0 {
+		logrus.Fatalln("Can't find any kafka pod")
+	}
 	params := &k8scmd.ExecOptions{
 		StreamOptions: k8scmd.StreamOptions{
 			Namespace:     ctlNamespace,

